packages/graph/hash_graph/lib/graph/src/api/rest/entity.rs

```diff
@@ -137,21 +132,39 @@ async fn get_entity<P: StorePool + Send>(
     tag = "Entity",
     responses(
         (status = 200, content_type = "application/json", description = "List of all entities", body = [PersistedEntity]),
+
         (status = 500, description = "Store error occurred"),
     )
 )]
 async fn get_latest_entities<P: StorePool + Send>(
     pool: Extension<Arc<P>>,
-    mut body: Option<Json<Expression>>,
 ) -> Result<Json<Vec<PersistedEntity>>, StatusCode> {
-    read_from_store(
-        pool.as_ref(),
-        &body
-            .take()
-            .map_or_else(Expression::for_latest_version, |json| json.0),
+    get_entities_by_query(pool, Json(Expression::for_latest_version())).await
+}
+
+#[utoipa::path(
+    get,
+    path = "/entities/{entityId}",
+    tag = "Entity",
+    responses(
+        (status = 200, content_type = "application/json", description = "The requested entity", body = PersistedEntity),
+        (status = 422, content_type = "text/plain", description = "Provided entity id is invalid"),
+
+        (status = 404, description = "entity was not found"),
+        (status = 500, description = "Datastore error occurred"),
```

Comment:
Just to flag this text is inconsistent with the other places (We call it Store) although I think all of the descriptions can be improved

Reply:
Applied a hotfix: 7d4b3ae1a8420c4398f817e293887dbf64067e31